update-db.js

```diff
@@ -65,6 +65,43 @@ function getLatestInfo (lock) {
   }
 }
 
+function getBrowsersList () {
+  return childProcess.execSync('npx browserslist').toString()
+    .trim()
+    .split('\n')
+    .map(function (line) { return line.trim().split(' ') })
+    .reduce(function (result, entry) {
+      if (!result[entry[0]]) {
+        result[entry[0]] = []
+      }
+      result[entry[0]].push(entry[1])
+      return result
+    }, {})
+}
+
+function diffBrowsersLists (old, current) {
+  return Object.keys(old)
```

Comment:
I also assume that the list of browsers itself won't change. e.g. if firefox was not in the old list it won't be in the current.
Though, my assumption can be wrong, since queries can be based on browsers usage statistics and they can change, so new browser can appear

Reply:
Fixed to remove this assumption